src/components/pages/newsletter.vue

```diff
@@ -4,14 +4,15 @@
       title="뉴스레터 아카이브"
     />
     <newsletter-list
-      :newsletters="newsletters"
+      :newsletters="newsletterList"
     />
   </div>
 </template>
 
 <script>
 import NewsletterList from '../organisms/newsletter/newsletter-list.vue';
 import SectionTitle from '../atoms/common/section-title';
+import { newsletter } from '../static/data/index.js';
```

Comment:
```suggestion
import { newsletter } from '../static/data';
```
여기도 index.js 를 숨기는 것을 제안드려 봅니다. 
보통 module 이 구성되면 해당 폴더만 import 하면 index.js 를 제일 먼저 찾아보게 되어있습니다.
이 규칙을 이용하면 index.js 를 코드상에서 숨기고 static/data 라는 폴더가 data 들을 가지고 있음을 알 수 있게 추상화된다고 생각합니다.

Reply:
넵 설명 감사합니다.